OpenStack-accounting/usr/local/sbin/manila_extract_accounting.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/python3
+import accountinglib
+
+import sys
+import time
+import datetime
+
+
+nowtime = time.localtime()
+
+print("Cinder Accounting run start")
```

Comment:
This should be `"Manila Accounting run start"`

Reply:
Fixed